src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -252,6 +267,10 @@ export class EditorComponent implements OnInit {
     this.maskCtx.drawImage(this.image, 0, 0);
 
     //  Only gets the image data from (0,0) to (width,height) of image.
+    this.originalImageData = this.maskCtx.getImageData(0, 0, imgWidth,
+      imgHeight
+    );
+    this.maskCtx.clearRect(0, 0, imgWidth, imgHeight);
     this.originalImageData = this.maskCtx.getImageData(0, 0, imgWidth, imgHeight);
     this.maskCtx.clearRect(0,0,imgWidth, imgHeight);
```

Comment:
Did something weird happen with the merging here? It looks like you have the same two lines repeated twice.